testkit/backend.py

```diff
@@ -3,8 +3,25 @@
 Assumes driver and backend has been built.
 Responsible for starting the test backend.
 """
-from common import run_in_driver_repo
+from common import open_proccess_in_driver_repo, is_browser
 import os
+import time
 
 if __name__ == "__main__":
-    run_in_driver_repo(["npm", "run", "start-testkit-backend"], env=os.environ)
+    print("starting backend")
+    if is_browser():
+        print("Testkit should test browser")
+        os.environ["TEST_ENVIRONMENT"] = "REMOTE"
+        print("openning firefox")
+
+    print("npm run start-testkit-backend")
+    with open_proccess_in_driver_repo([
+            "npm", "run", "start-testkit-backend"
+    ], env=os.environ) as backend:
+        if (is_browser()):
+            time.sleep(5)
+            with open_proccess_in_driver_repo([
```

Comment:
```suggestion

    print("npm run start-testkit-backend")
    with open_proccess_in_driver_repo([
        "npm", "run", "start-testkit-backend"
    ], env=os.environ) as backend:
        if (is_browser()):
            time.sleep(5)
            print("openning firefox")
            with open_proccess_in_driver_repo([
```

Main change is: print "openning firefox" before it actually happens.